sklearn/cluster/mean_shift_.py

```diff
@@ -113,6 +124,13 @@ def mean_shift(X, bandwidth=None, seeds=None, bin_seeding=False,
     See examples/cluster/plot_meanshift.py for an example.
 
     """
+    # FIXME To be removed in 0.18
+    if max_iterations is not None:
+        warnings.warn("The `max_iterations` parameter has been renamed to "
+                      "`max_iter`. The `max_iterations` parameter will be "
+                      "removed in 0.18", DeprecationWarning)
```

Comment:
When deprecation happen it's good to state both the version when it happened (in this case it will be the next release, 0.16) and the version when it will be removed (0.18 as you stated here).